app/models/action.rb

```diff
@@ -1,6 +1,10 @@
 class Action < ActiveRecord::Base
   belongs_to :page
   belongs_to :action_user
+  after_create :update_page_action_count
 
+  def update_page_action_count
+    self.page.action_count = self.page.actions.count
```

Comment:
Eric I don't really understand that at all. Is there any case in which we allow a user to do multiple actions on a page?

Also, incrementing and copying the count after create should be functionally identical because creating an action inherently increments `page.action.count`, doesn't it? You're worried about page.action_count getting out of sync somehow?